layouts/default.vue

```diff
@@ -198,18 +314,35 @@
 </script>
 
 <style>
-	.notification p {
+	.notification-content-msg p {
 		margin-bottom: 18px;
+		word-break: break-all;
 	}
 
-	.notification p:last-child {
+	.notification-content-msg p strong {
+		word-break: break-all;
+	}
+
+	.notification-content-msg p:last-child {
 		margin-bottom: 0;
 	}
 
-	.notification a {
+	.notification-content-msg a {
 		color: var(--p-primary-color);
 		font-weight: 600;
 	}
+
+	.n-popover[data-pc-name="popover"] {
+		@apply !rounded-xl;
+		@apply !overflow-hidden;
+		@apply absolute !ml-4 !mt-2;
+	}
+
+	.n-popover[data-pc-name="popover"] > *[data-pc-section="content"] {
+		@apply flex items-center;
+		@apply !rounded-xl;
+		@apply border dark:border-[var(--table-border)];
+	}
```

Comment:
Here it can be done too via `:pt={ content: { class: '...' } }` on the popover component